client/tests/test_client.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from client import models
+from client.models import Client
+from django.core.exceptions import ValidationError
+
+
+USER_NAME_TEST = 'liorsil'
+FIRST_NAME_TEST = 'lior'
+LAST_NAME_TEST = 'silber'
+PASSWORD_TEST = '1234qwer'
+USER_TYPE_TEST = 'client'
+AREA_TEST = 'yuvalim'
+
+
+@pytest.fixture
+def client0():
+    return models.Client(user_name=USER_NAME_TEST,
+                         first_name=FIRST_NAME_TEST,
+                         last_name=LAST_NAME_TEST,
+                         password=PASSWORD_TEST,
+                         user_type=USER_TYPE_TEST,
+                         area=AREA_TEST)
+
+
+@pytest.fixture
+def saved_client0(client0):
+    client0.save()
+    return client0
+
+
+class TestClientModel:
+    @pytest.mark.django_db()
+    def test_save_client(self, client0):
+        """Test - checks if client is saved in DB.
+        Returns:
+            None.
+        """
+        Client.save_client(client0)
+        assert client0 in models.Client.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_client(self, saved_client0):
+        """Test - checks if client is not in DB after deletion.
+        Returns:
+            None.
+        """
+        Client.delete_client(saved_client0)
+        assert client0 not in Client.objects.all()
+
+    @pytest.mark.django_db()
+    def test_get_all_clients(self):
+        """Test - checks if All clients are found in the DB.
+        Args:
+            None.
+        Returns:
+            None.
+        """
+        for cli in Client.get_all_clients():
```

Comment:
Do you actually have any clients there? Maybe verify that `Client.objects.all()` is not empty before.